### PR TITLE
Switched to PSR-2

### PR DESCRIPTION
**Pull request by lolautruche**
+ added config file for PHP-CS-Fixer

### CODE REVIEW

**Comment by andrerom** [comment]
Seems ezrobot is missing fore this repo, add and force push to check?

**Comment by lolautruche** [comment]
It's not configured on this repo

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        '-concat_without_spaces',
+        '-empty_return',
+        '-phpdoc_params',
+        '-phpdoc_separation',
+        '-phpdoc_to_comment',
+        '-spaces_cast',
+        '-blankline_after_open_tag',
+        '-single_blank_line_before_namespace',
+        // psr0 has weird issues with our PSR-4 layout, so deactivating it.
+        '-psr0',
```
Comment (michalpipa) [inline .php_cs]: I usually enable this check on Studio bundles. I didn't have any problems so far.

Comment (lolautruche) [inline .php_cs]: It's because of split with `lib/` and `mvc`. If you don't have that, then no problem

**Comment by andrerom** [comment]
+1

**Pull request merged**